src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -149,8 +150,52 @@ export const OpportunityCard = ({
     <>
       {loginModalOpen && <LoginModal onClose={() => setLoginModalOpen(false)} isOpen={loginModalOpen} />}
       <Component className="flex flex-col bg-white p-5 sm:p-6 rounded shadow-border">
+        <div className="order-2 flex flex-col">
+          <span className="font-arial text-body-sm-mobile sm:text-body-sm leading-6 uppercase">{cardTypeTitle}</span>
+          <NavLink
+            to={to}
+            className="mb-2 text-heading-2-mobile sm:text-heading-2 hyphens-auto hover:underline hover:text-link"
+          >
+            {name}
+          </NavLink>
+          <p className="font-arial text-body-md-mobile sm:text-body-md">{description}</p>
+          <div className="flex flex-wrap mt-5">
+            <BottomBox title={t('opportunity-card.trend')} className="bg-todo">
+              {trend === 'NOUSEVA' ? (
+                <MdOutlineTrendingUp size={24} className="text-accent" aria-label={t(`opportunity-card.trend-up`)} />
+              ) : (
+                <MdOutlineTrendingDown
+                  size={24}
+                  className="text-accent"
+                  aria-label={t(`opportunity-card.trend-down`)}
+                />
+              )}
+            </BottomBox>
+            <BottomBox title={t('opportunity-card.employment-outlook')} className="bg-todo">
+              <OutlookDots
+                outlook={employmentOutlook}
+                ariaLabel={t('opportunity-card.outlook-value', { outlook: employmentOutlook })}
+              />
+            </BottomBox>
+            {hasRestrictions && (
+              <BottomBox title={t('opportunity-card.maybe-has-restrictions')} className="bg-todo">
+                <MdBlock className="text-accent" size={20} role="presentation" />
```

Comment:
`role="presentation"` added for icon as the whole **BottomBox** in this case  is only shown when it has restrictions and the title already holds the needed information.